frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -385,7 +391,10 @@ const AssociatedLandForm: React.FC<IAssociatedLandForm> = (props: IAssociatedLan
                     if (currentProgess < 15) {
                       incrementProgress();
                     } else {
-                      const parcels = [...Array(numParcels)].map(n => getInitialLandValues());
+                      const parcels = [...Array(numParcels)].map(n => ({
+                        ...getInitialLandValues(),
+                        agencyId: keycloak.agencyId,
```

Comment:
by default the agency should be the users agency.

Reply:
Just make sure this doesn't allow a user to change the agency of a property.  The API should only allow `property-admin` to change the owner of an agency through this form.  Normal users will always be forced to only add property to their agency.

Reply:
yes, this is just initializing the initial state of the parcels, when the user searches for pre-existing parcels it'll use whatever agency is returned by PIMS.